semantic-kernel-python/run.sh

```diff
@@ -7,42 +7,28 @@ set -x
 CONN_STRING=$(fetch_local_atlas_uri)
 PYTHON_BINARY=$(find_python3)
 
-
 # WORKING_DIR = src/semantic-kernel-python/semantic-kernel
 cd python
 
+$PYTHON_BINARY -m venv .
+source ./bin/activate
 
-# Install Poetry into base python
-$PYTHON_BINARY -m pip install -U pip poetry
-# Create a package specific poetry environment
-$PYTHON_BINARY -m poetry env use $PYTHON_BINARY
-# Activate the poetry env, which itself does not include poetry
-source $($PYTHON_BINARY -m poetry env info --path)/bin/activate
-# Recreate the poetry lock file
-$PYTHON_BINARY -m poetry lock --no-update
-# Install from pyproject.toml into package specific environment
-$PYTHON_BINARY -m poetry install --with dev
-
-# Create .env file
-python -c "from dotenv import set_key; set_key('.env', 'OPENAI_API_KEY', '$openai_api_key')"
-python -c "from dotenv import set_key; set_key('.env', 'MONGODB_ATLAS_CONNECTION_STRING', '$CONN_STRING')"
+make install
```

Comment:
Do you also have to run `make build`?

Reply:
Followed the [dev instructions here](https://github.com/microsoft/semantic-kernel/blob/main/python/DEV_SETUP.md#for-mac-and-linux-both-native-and-wsl) and I found I did not have to!